Q13.js

```diff
@@ -0,0 +1,4 @@
+//Q-13: Create two variables -- one to hold the names of the kittens and one to hold the names of the adult cats. Make sure to choose an appropriate data type to hold this information!
```

Comment:
Since the information we want to store is a list of things (names of cats), we should use an array (with the square brackets `[]`).
```javascript
let nameAdult = ['Mezzo','Sabrina','Josie'];
let nameKitten = ['Kyle Meowry','Luna','Obi','Charlie','Raspberries'];
```